guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,73 @@
+[[migrating_pulp_content]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content and the number of content views.
+For large systems, this can mean several days of downtime.
+To prevent this, {Team} recommends that you pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
+This reduces the overall upgrade downtime.
+
+Use this procedure to begin migrating content from Pulp 2 to Pulp 3.
+
+.Procedure
+
+. View details of the content you are migrating using the following command:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content migration-stats
+----
++
+Use this command as often as necessary during the migration process to determine how long the process will take.
+It also identifies corrupted or missing content that might cause the migration to fail.
+Output is similar to the following:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+Running Retrieve Pulp 2 to Pulp 3 migration statistics
+============================================
+Retrieve Pulp 2 to Pulp 3 migration statistics:
+============Migration Summary================
+Migrated/Total RPMs: 0/367633
+Migrated/Total errata: 0/20780140
+Migrated/Total repositories: 0/33924
+Estimated migration time based on yum content: 47 hours, 23 minutes
+Note: ensure there is sufficient storage space for /var/lib/pulp/published to
+double in size before starting the migration process.
+Check the size of /var/lib/pulp/published with 'du -sh /var/lib/pulp/published/'
+[OK]
+----
+
+. Update the file permissions before the {ProjectServer} upgrade using the following:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} prep-{TargetVersion}-upgrade
+----
++
+This might take some time on high latency systems.
+. Prepare your content for Pulp migration using the following command:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content-prepare
+----
++
+You cannot use `*Ctrl*` *+* `*C*` to terminate this process.
```

Comment:
I think if you tell someone that you cannot use ctrl+C, you have to tell them what to do in case they do accidently use it :)

"If a user does terminate it with ctrl-c or with a ssh disconnect, the process will continue int he background, but can be stopped by running the 'content-prepare abort' command"

Reply:
@ianf77 please confirm the text change for me here and I will apply it.

Reply:
If a user does attempt to halt the process with *CTRL* *+* *C* or with a SSH disconnect, the process does not terminate but continues in the background. 
Followed by the You can use... line

Reply:
Fixed